code/modules/mob/living/carbon/xenomorph/facehuggers.dm

```diff
@@ -338,14 +338,27 @@
 
 	if(M.status_flags & XENO_HOST || isxeno(M))
 		return FALSE
-	if(!self_done)
-		M.visible_message("<span class='danger'>[src] leaps at [M]'s face!</span>")
 
 	if(isxeno(loc)) //Being carried? Drop it
 		var/mob/living/carbon/Xenomorph/X = loc
 		X.dropItemToGround(src)
 		X.update_icons()
 
+	if(M.in_throw_mode && M.dir != dir && !M.incapacitated() && !M.get_active_held_item())
+		var/catch_chance = 50
+		if(M.dir == reverse_dir[dir])
+			catch_chance += 20
+		if(M.lying)
+			catch_chance -= 50
```

Comment:
Chatching chance for a lying mob should be zero. Else you'll get nested hosts catching huggers.

Reply:
Nested hosts are usually restrained/incapacitated and so can't do jack.
I were merely thinking about /tg/'s mobility code and crawling when letting this be, in the case we ever get there.

Reply:
Hmm, fair enough.